app/actions/remote/channel.ts

```diff
@@ -920,66 +932,7 @@ export async function getChannelTimezones(serverUrl: string, channelId: string)
     }
 }
 
-export async function getOrCreateDirectChannel(serverUrl: string, otherUserId: string, shouldSwitchToChannel = true) {
```

Comment:
This function was doing practically the same as makeDirectChannel, and only used by the handleGoToLocation function.